src/include/liburing.h

```diff
@@ -499,6 +499,7 @@ IOURINGINLINE void io_uring_prep_recvmsg_multishot(struct io_uring_sqe *sqe,
 						   unsigned flags)
 {
 	io_uring_prep_recvmsg(sqe, fd, msg, flags);
+	sqe->flags |= IOSQE_BUFFER_SELECT;
```

Comment:
These get removed in the previous commit and then re-added, why?

Reply:
In my first commit, I thought it's a good idea to re-add the flag. Later thought it could be better to just do it with an initialization function. So first add, then remove.

The order of two commits might have been messed up. Because I had to force push to get the commit message right.

The final version of my pull request doesn't have these re-added flags.